### DISCUSSION TITLE
Error with profile

### DISCUSSION THREAD

**Discussion by SteliosGee** [community/community#148255] [Mobile] [answered]
Since I downloaded the app, everytime I try to check my profile through the mobile app it throws an error. ![Screenshot_2024-12-31-22-02-10-456_com.github.android.jpg](https://github.com/user-attachments/assets/849f2731-2048-423d-ab7a-5ea30e8a59d6)

**Comment by ibra-kdbra** [comment]
Are you in a restricted area(zone), if not contact the support--> this will take sometime.

A quick fix can be by connecting with _VPN_, **secured VPN** to be precise.

**Reply by SteliosGee** [reply, marked as answer]
I fixed it by clearing all data and login again!

**Reply by ibra-kdbra** [reply]
Okay, good for you, then make this question as **completed**.
